### PR TITLE
New Crowdin updates

### LINKED ISSUE #185: Setup i18n with Crowdin
- [x] Create a Crowdin project : https://crowdin.com/project/ferdium-app
- [x] Setup the Crowdin → GitHub integration to submit daily PRs 
- [x] Fix the internationalization setup

The setup is a bit strange for i18n at the moment:
- The code languages listed in `src/i18n/locales` are not consistent: some are two characters locales code like `fr.json`, other are regionalized locale codes like `zh-HANT.json` or `en-US.json`
- The languages listed here in `src/i18n/languages.ts` (availables to the users) doesn't correspond to the languages listed in Crowdin, nor to the files present in the `src/i18n/locales/` folder. For instance:
    - `zh-HANT` is meant to be Chinese Traditional according to [some sources](https://www.alchemysoftware.com/livedocs/ezscript/Topics/Catalyst/Language.htm), but it's mapped to Chinese Simplified on Crowdin.
    - ~~"Nederlands" is marked in the UI, but it is "Dutch" on Crowdin.~~ expected as we write the native language names in the UI
    - Vietnamese is translated to 90% but is not available to the users. (fixed by #562)
    
We have decided to not hold the 6.0.0 release for this, as the current translations are working. But this will be fixed before allowing translations again through Crowdin.

If you see some strings that need to be fixed before 6.0.0, please report them below this issue.

### PR DESCRIPTION
**Pull request by vraravam**
Will close #185.

This came with some changes to the Crowdin config, which is now the same as in the crowdin.yml file in this repo. 
`skip_untranslated_strings` has been activated to reduce the file size with "useless" non-translations (our i18n tool handle this in the UI, with a fallback with the default English string)

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 files:
   - source: /src/i18n/locales/en-US.json
     translation: /src/i18n/locales/%two_letters_code%.json
+commit_message: "[ci skip]"
```
Comment (vraravam) [inline crowdin.yml]: should this be `[skip ci]` instead? i have always used this - but, if `[ci skip]` works, then great!

Comment (nathanaelhoun) [reply crowdin.yml]:
```suggestion
commit_message: "[skip ci]"
```
You are right

**Comment by nathanaelhoun** [comment]
Ok, they are still some inconsistencies between langages codes referenced in Crowdin and those listed in [`languages.ts`](https://github.com/ferdium/ferdium-app/blob/develop/src/i18n/languages.ts), I'll look at them in the next few days

**Comment by SpecialAro** [comment]
@nathanaelhoun what file is this that was added [src/i18n/locales/pr-BR.json](https://github.com/ferdium/ferdium-app/pull/133/files#diff-7afb9bf7ed8f5870214658ff6a5628a384356c674b67c0751fc7cf2b6b545030) ? Is it supposed to be Portuguese (Brazil) ?

If yes, it should be pt-BR

**Comment by nathanaelhoun** [comment]
> @nathanaelhoun what file is this that was added [src/i18n/locales/pr-BR.json](https://github.com/ferdium/ferdium-app/pull/133/files#diff-7afb9bf7ed8f5870214658ff6a5628a384356c674b67c0751fc7cf2b6b545030) ? Is it supposed to be Portuguese (Brazil) ?
> 
> If yes, it should be pt-BR

Yes, I made a typo 😅
I will check on the i18n configuration later this week-end, I'm currently moving and my desk is disassembled

**Review by nathanaelhoun — CHANGES_REQUESTED** [review]
Configuration messed up, please do not merge ATM

**Comment by vraravam** [comment]
@nathanaelhoun - can we merge this please?

**Comment by nathanaelhoun** [comment]
@vraravam Issue #185 created to track the remaining tasks for this PR. It can wait 6.0.0 release.

**Review by nathanaelhoun — APPROVED** [review]
now it seems ok to me

**Pull request merged**